definitions/mocha_v2.4.x/flow_>=v0.22.x/mocha_v2.4.x.js

```diff
@@ -0,0 +1,19 @@
+type TestFunction = (() => void | Promise) | ((done : Function) => void);
+
+declare class describe {
```

Comment:
Since `describe` is not meant to be `new`-ed, this would be better described as a callable object:

```javascript
declare var describe: {
  (name: string, spec: () => void): void;
  ...
};
```